### PR TITLE
HAAR-858: Removing the splash screen before sign-in

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: public/javascripts/adminBundle.js [hunk 1]
```diff
@@ -38,14 +38,14 @@ module.exports = jQuery;
 var __webpack_exports__ = {};
 // This entry need to be wrapped in an IIFE because it need to be isolated against other modules in the chunk.
 (() => {
-var $ = __webpack_require__(1);
+const $ = __webpack_require__(1);
```
Comment (brightonsbox) [inline public/javascripts/adminBundle.js:41]: Changes autogenerated as part of webpack upgrade

File: public/javascripts/morelessBundle.js [hunk 2]
```diff
@@ -38,13 +38,13 @@ module.exports = jQuery;
 var __webpack_exports__ = {};
 // This entry need to be wrapped in an IIFE because it need to be isolated against other entry modules.
 (() => {
-var $ = __webpack_require__(1);
+const $ = __webpack_require__(1);
```
Comment (brightonsbox) [inline public/javascripts/morelessBundle.js:41]: Changes autogenerated as part of webpack upgrade

File: public/javascripts/tabs.js [hunk 3]
```diff
@@ -12,28 +12,28 @@ module.exports = jQuery;
 /* 3 */
 /***/ ((__unused_webpack_module, exports, __webpack_require__) => {
 
-var $ = __webpack_require__(1);
+const $ = __webpack_require__(1);
```
Comment (brightonsbox) [inline public/javascripts/tabs.js:15]: Changes autogenerated as part of webpack upgrade

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**